src/plugins/intel_cpu/tests/unit/shape_inference_test/batch_norm_inference_shape_inference_test.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "openvino/op/batch_norm.hpp"
+#include "utils.hpp"
+
+using namespace ov;
+using namespace ov::intel_cpu;
+using namespace testing;
+
+class BatchNormInferenceV0StaticShapeInferenceTest : public OpStaticShapeInferenceTest<op::v0::BatchNormInference> {
+protected:
+    void SetUp() override {
+        output_shapes.resize(1);
+    }
+};
+
+TEST_F(BatchNormInferenceV0StaticShapeInferenceTest, default_ctor) {
+    const auto op = make_op();
+
+    input_shapes = {{3, 5, 7}, {5, 7}, {5, 7}, {5, 7}};
+    output_shapes = shape_inference(op.get(), input_shapes);
```

Comment:
Call `shape_infer` directly because if new inferences are note added this helper will not use static shapes.

Make note that this function should be used when new inferences added to CPU shape inference factory.

Reply:
Done.